package.json

```diff
@@ -90,6 +90,7 @@
     "chai": "^4.1.2",
     "cross-env": "^5.0.5",
     "cssnano": "^3.10.0",
+    "cytoscape-popper": "git+https://github.com/cytoscape/cytoscape.js-popper.git",
```

Comment:
Maybe it is time to release the popper extension.  @maxkfranz what do you think?

Reply:
It still needs some work.  I haven't had a chance to review to make issues for it yet.